src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img
+        src="images/imac.jpeg"
+        alt="picture of Apple device"
+        class="card__picture"
+        >
+      </div>
+      <h3 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K
+        (MNED2UA/A)
+      </h3>
+      <p class="card__cod">Код товара: 195434</p>
+      <div class="card__rating">
+        <div class="card__stars card__stars--stars--4">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
```

Comment:
Тут казали вставляти stars блоком з минулого завдання. Тобто окрема зірка повинна бути елементом блоку з набором зірок, а не картки напряму.
```suggestion
        <div class="card__stars stars--stars-4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```